server/db/postgres/dataGenerator.js

```diff
@@ -37,22 +37,26 @@ log('symbol generation complete');
 //   find a random company name
 //   add element and company name to query string
 // send query
-var queriesArray = [];
```

Comment:
Recommend removing 'array' and data structure types from variable names, I've read that's not good practice.  By pluralizing a variable, it tells the reader it's a collection like an array.